### DISCUSSION TITLE
PRs

### DISCUSSION THREAD

**Discussion by lloydbayley** [Nachtzuster/BirdNET-Pi#176] [General]
There seems to be/have-been a lot of PR's lately with little or no description. Some merged, some not. How about we make sure there are adequate descriptions so that things are easier to follow?

**Comment by Emmo213** [comment]
Since the only recent PRs without descriptions are mine I'll assume I was the target for this comment. 

If we need a descriptions even when referencing other discussions/issues that's fine, I can do that, but it's going to basically be the same info as whatever the PR is referencing. For instance PR #171  is named "Issue #170 - set timezone on the todays detection page" because it fixes issue #170. Similarly PR #166 is the enhancement for #165 so it's named "Feature #165 Wiki link in notification".

@Nachtzuster what standard do you want for opening a PR?
